docs/user-guide/ze-profiles.md

```diff
@@ -110,60 +125,74 @@ You can use the sample to customize your profile configuration file. Ensure that
+To create a `zosmf` compatible profile:
 
 1. Navigate to the explorer tree.
 2. Click the **+** button next to the **DATA SETS**, **USS** or **JOBS** bar.
 
-   **Note:** If you already have a profile, select it from the drop-down menu.
+  :::note
+
+  If you already have a profile, select it from the drop-down menu in the **picker** field.
+
+  :::
 
 3. Select the **Create a New Connection to z/OS** option.
 
-   **Note:** When you create a new profile, user name and password fields are optional. However, the system will prompt you to specify your credentials when you use the new profile for the first time.
+   :::note
+
+   When you create a new profile, user name and password fields are optional. However, the system will prompt you to specify your credentials when you use the new profile for the first time.
+
+   :::
 
 4. Follow the instructions, and enter all required information to complete the profile creation.
 
-<img src={require("../images/ze/ZE-newProfiles.gif").default} width="600" alt="New Connection"/>
+  <img src={require("../images/ze/ZE-newProfiles.gif").default} width="600" alt="New Connection"/>
 
-You successfully created a Zowe CLI `zosmf` profile. Now you can use all the functionalities of the extension.
+  You successfully created a Zowe CLI `zosmf` profile. Now you can use all the functionalities of the extension.
 
-If you need to edit a profile, right-click the profile and select **Update Profile** option.
+To edit a profile:
 
-<img src={require("../images/ze/ZE-prof-update.gif").default} width="600" height="300" alt="Edit a Profile"/>
+1. Right-click the profile and select **Update Profile** option.
 
-In addition, you can hide a profile from the explorer tree, and permanently delete a profile. When you delete your profile permanently, the extension erases the profile from the `.zowe` folder. To hide or delete a profile, right-click the profile and choose one of the respective options from the list.
+2. Edit the profile information in the **picker** field.
 
-### Validating profiles
+    <img src={require("../images/ze/ZE-prof-update.gif").default} width="600" height="300" alt="Edit a Profile"/>
 
-**Note:** The following information applies to Zowe CLI V1 profiles (one yaml file for each user profile) and Zowe CLI team profiles (Zowe CLI V2).
+To hide a profile from the tree view, right-click the profile and select the **Hide Profile** option.
 
-Zowe Explorer includes the profile validation feature that helps to ensure that z/OSMF is accessible and ready for use. If a profile is valid, the profile is active and can be used. By default, the feature is automatically enabled. You can disable the feature by right-clicking on your profile and selecting the **Disable Validation for Profile** option. Alternatively, you can enable or disable the feature for all profiles in the VS Code settings.
+To delete a profile from the tree view, right-click the profile and select the **Delete Profile** option.
```

Comment:
This does not only remove the profile from the tree view, but also deletes the profile from the system, 
i.e. `~/.zowe/profiles/<type>/myDeletedProfile.yaml`